lib/date.ts

```
@@ -0,0 +1,20 @@
+export function formatDateTime(date: Date): string {
+  return date.toISOString().replace("T", " ").substring(0, 19)
+}
+
+/**
+ * Format to ClickHouse-compatible `DateTime64(3)` format
+ * Format: "YYYY-MM-DD HH:mm:ss.SSS"
+ */
+export function formatDateTime64(date: Date): string {
+  const iso = date.toISOString()
+  const [datePart, timePart] = iso.split("T")
+  const [hhmmss, ms] = timePart?.split(".")
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)